contracts/protocol/bases/SellerBase.sol

```diff
@@ -49,33 +49,36 @@ contract SellerBase is ProtocolBase, IBosonAccountEvents {
             ADMIN_OR_AUTH_TOKEN
         );
 
-        // Check that the addresses are unique to one seller id, across all roles. These addresses should always be checked. Treasury is not checked
-        mapping(address => uint256) storage sellerIdByOperator = lookups.sellerIdByOperator;
-        mapping(address => uint256) storage sellerIdByAdmin = lookups.sellerIdByAdmin;
-        mapping(address => uint256) storage sellerIdByClerk = lookups.sellerIdByClerk;
-        require(
-            sellerIdByOperator[_seller.operator] == 0 &&
-                sellerIdByOperator[_seller.clerk] == 0 &&
-                sellerIdByAdmin[_seller.operator] == 0 &&
-                sellerIdByAdmin[_seller.clerk] == 0 &&
-                sellerIdByClerk[_seller.operator] == 0 &&
-                sellerIdByClerk[_seller.clerk] == 0,
-            SELLER_ADDRESS_MUST_BE_UNIQUE
-        );
```

Comment:
I believe we still have to check:

```javascript
require(sellerIIdByClerk[sender] = 0 && sellerIdByOperator[sender] = 0
```

otherwise, we won't be validating when is auth type